pkgs/python_ot/pyproject.toml

```diff
@@ -95,9 +95,9 @@ wcwidth = "0.2.6"
 wheel = "0.41.2"
 yapf = "0.32.0"
 zipfile2 = "0.0.12"
-chipwhisperer = { git = "https://github.com/newaetech/chipwhisperer-minimal.git", rev = "2643131b71e528791446ee1bab7359120288f4ab" }
-edalize = { git = "https://github.com/lowRISC/edalize.git", tag = "v0.4.0"}
-fusesoc = { git = "https://github.com/lowRISC/fusesoc.git", tag = "ot-0.4" }
+chipwhisperer = { url = "https://github.com/newaetech/chipwhisperer-minimal/archive/2643131b71e528791446ee1bab7359120288f4ab.zip" }
+edalize = { url = "https://github.com/lowRISC/edalize/archive/refs/tags/v0.4.0.zip" }
+fusesoc = { url = "https://github.com/lowRISC/fusesoc/archive/refs/tags/ot-0.5.zip" }
```

Comment:
Is `url` recommended over `git`?

Reply:
pip can't pin git dependencies with hash, so OT currently use url dependency.